src/TypeScript/EslintBridgeClient/EslintBridgeProcess.cs

```diff
@@ -137,8 +140,11 @@ private void StartServer()
             Process.OutputDataReceived += OnOutputDataReceived;
 
             Process.Start();
-            logger.WriteLine(Resources.INFO_ServerProcessId, Process.Id);
-            logger.LogDebug(Resources.INFO_ServerHasExisted, Process.HasExited);
+            // Storing the process so we don't have to worry about checking whether Process is null when logging,
+            // and so that if any messages are logged after Process is set to null we know which process it related to.
+            processId = Process.Id;
+            logger.WriteLine(Resources.INFO_ServerProcessId, processId);
+            logger.LogDebug($"[eslint-bridge] [process id: {processId}] Server process HasExited: {1}.", processId, Process.HasExited);
```

Comment:
All of the messages start (debug and non-debug) now show the process id at the start of the message in this format:
```[prefix] [process id: {processId}] Message```

Feel free to suggest an alternative.

I considered omitting the `process id: ` label, but left it in so there was would be no confusion about whether it was logging the port number or the process id.

I also considered logging the port number, but we won't have it until some time after the process has started and we've received the output stream message from the node process.